### PR TITLE
A few new maint lootspawner additions

### PR DESCRIPTION
**Pull request by Bawhoppen**
inb4 I am accused of powercreep

### CODE REVIEW

**Comment by Zilenan91** [comment]
Can you add some more maint loot spawners in general to Box?

**Comment by Bawhoppen** [comment]
@Zilenan91 I'd love to but map freeze, blame @duncathan

**Comment by Niknakflak** [comment]
Excuse me, we do not allow player references in code

**Comment by Bawhoppen** [comment]
no player references here

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -120,6 +120,26 @@
...
+				/obj/item/weapon/grenade/smokebomb = 2,
+				/obj/item/device/geiger_counter = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/grown/citrus/orange = 1, //only for the best
+				/obj/item/device/radio/headset = 1,
+				/obj/item/device/assembly/infra = 1,
+				/obj/item/device/assembly/igniter = 2,
+				/obj/item/device/assembly/signaler = 2,
+				/obj/item/device/assembly/mousetrap = 2,
+				/obj/item/weapon/reagent_containers/syringe = 2,
+				/obj/item/clothing/gloves/color/orange = 1,
```
Comment (Cheridan) [inline code/game/objects/effects/spawners/lootdrop.dm]: can't you make a gloves/color/random?

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 2]
```diff
@@ -120,6 +120,27 @@
 				/obj/item/weapon/wrench = 4,
 				/obj/item/weapon/relic = 3,
 				/obj/item/weaponcrafting/reciever = 1,
+				/obj/item/clothing/head/cone = 2,
+				/obj/item/weapon/skub = 1,
+				/obj/item/weapon/grenade/smokebomb = 2,
+				/obj/item/device/geiger_counter = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/grown/citrus/orange = 1, //only for the best
```
Comment (Niknakflak) [inline code/game/objects/effects/spawners/lootdrop.dm]: excuse me player references aren't allowed

File: code/game/objects/effects/spawners/lootdrop.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/game/objects/effects/spawners/lootdrop.dm]: ok I will

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 3]
```diff
@@ -120,6 +120,27 @@
...
+				/obj/item/clothing/gloves/color/orange = 1,
+				/obj/item/clothing/gloves/color/red = 1,
+				/obj/item/clothing/gloves/color/blue = 1,
+				/obj/item/clothing/gloves/color/purple = 1,
+				/obj/item/clothing/gloves/color/green = 1,
+				/obj/item/clothing/gloves/color/grey = 1,
+				/obj/item/clothing/gloves/color/light_brown = 1,
+				/obj/item/clothing/gloves/color/brown = 1,
+				/obj/item/clothing/shoes/laceup = 4,
+				/obj/item/weapon/reagent_containers/food/condiment/saltshaker = 10, //added for impartiality
```
Comment (Niknakflak) [inline code/game/objects/effects/spawners/lootdrop.dm]: Sorry baw, you can't reference yourself either

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 4]
```diff
@@ -120,6 +120,27 @@
 				/obj/item/weapon/wrench = 4,
 				/obj/item/weapon/relic = 3,
 				/obj/item/weaponcrafting/reciever = 1,
+				/obj/item/clothing/head/cone = 2,
+				/obj/item/weapon/skub = 1,
```
Comment (KazeEspada) [inline code/game/objects/effects/spawners/lootdrop.dm]: ANTI-SKUB!

Comment (PKPenguin321) [inline code/game/objects/effects/spawners/lootdrop.dm]: fuck yoooou

File: code/game/objects/effects/spawners/lootdrop.dm (hunk 3 above)
Comment (xxalpha) [inline code/game/objects/effects/spawners/lootdrop.dm]: hue

**Comment by Impisi** [comment]
Finally now people can't meta traitors for wearing laceup shoes.

I agree that some more drop points would be great for balancing the table.

**Comment by Bawhoppen** [comment]
@Impisi lawyers have laceups in their locker, you're way behind the times

**Comment by Zilenan91** [comment]
Bartenders also spawn in lace-ups but any self respecting bartender puts on the black shoes

**Comment by ShadowDeath6** [comment]
phat lute

:+1: Pretty cool

**Comment by PKPenguin321** [comment]
Merge it do it just do it don't let your memes dream etc.

**Comment by newfren** [comment]
> Bartenders also spawn in lace-ups but any self respecting bartender puts on the black shoes

You're scum.

File: code/game/objects/effects/spawners/lootdrop.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/game/objects/effects/spawners/lootdrop.dm]: ok so nobody can give me a straight answer how to do that I guess I'm not for now

**Comment by duncathan** [comment]
Didn't I help you with the gloves @Bawhoppen

**Comment by Bawhoppen** [comment]
yes but it wasn't what I was hoping for, I can do it I guess but its 0/10

File: code/game/objects/effects/spawners/lootdrop.dm (hunk 1 above)
Comment (Dorsisdwarf) [inline code/game/objects/effects/spawners/lootdrop.dm]: Isn't gloves/color/random a path|?

**Pull request merged**